app/src/main/java/org/stepic/droid/storage/operations/DatabaseFacade.kt

```diff
@@ -170,6 +170,10 @@ constructor(
     fun addLessons(lessons: List<Lesson>) =
         lessonDao.insertOrReplaceAll(lessons)
 
+    fun removeLessons(courseId: Long) {
+        lessonDao.removeAllInRange(DbStructureLesson.Columns.COURSES, courseId.toString())
```

Comment:
- Will it work for lessons with multiple courses?
- Will it work if we have lesson with `courses=[6700]` and lesson with `courses=[67]` and we want to remove only second one by passing `courseId=67`?